amqp_influxdb/__main__.py

```diff
@@ -47,10 +55,24 @@ def main():
         batch_size=args.influx_batch_size,
         max_batch_delay=args.influx_max_batch_delay)
 
+    if args.amqp_port != '':
+        amqp_port = args.amqp_port
+    else:
+        if args.amqp_ca_cert_path != '':
```

Comment:
If we indeed add the ssl_enabled setting I mentioned above, it would also make sense to use it here and not refer to ca_cert_path to deduce the ssl mode.
We could also use consts have the clear condition:
if args.ssl_enabled:
    amqp_port = AMQP_SSL_PORT
else:
    amqp_port = AMQP_DEFAULT_PORT